src/redux/checkout/tests/checkout.actions.test.js

```diff
@@ -241,7 +257,356 @@ describe('get prices from NovaPoshta API', () => {
     expect(getNovaPoshtaPrices(payload)).toBeTruthy();
   });
 
-  it('value by key "type" of returned object should to be equal to "GET_NOVAPOSHTA_PRICES"', () => {
+  it('value by key "type" of returned object should be equal to "GET_NOVAPOSHTA_PRICES"', () => {
     expect(getNovaPoshtaPrices(payload).type).toEqual(type);
   });
 });
+
+// regions from UkrPoshta action tests
+describe('set regions from UkrPoshta action test', () => {
+  let type;
+  let setUPRegionsAction;
+
+  const payload = [
+    {
+      REGION_ID: '1',
+      REGION_UA: 'Вінницька',
+      REGION_EN: 'Vinnytska'
+    },
+    {
+      REGION_ID: '2',
+      REGION_UA: 'Волинська',
+      REGION_EN: 'Volynska'
+    }
+  ];
+
+  beforeEach(() => {
+    type = SET_UKRPOST_REGIONS;
+    setUPRegionsAction = setUkrPostRegions(payload);
+  });
+
+  it('should return object', () => {
+    expect(typeof setUPRegionsAction).toStrictEqual('object');
+  });
+
+  it('should be truthy', () => {
+    expect(setUPRegionsAction).toBeTruthy();
+  });
+
+  it('value by key "type" of returned object should be equal to "SET_UKRPOST_REGIONS"', () => {
+    expect(setUPRegionsAction.type).toEqual(type);
+  });
+
+  it('value by key "payload[0].REGION_UA" of returned object should be equal to "Вінницька"', () => {
+    expect(payload[0].REGION_UA).toEqual('Вінницька');
```

Comment:
get this string from variable as const

Reply:
Makes no sense if we import the same data into two different variables and then compare them to be equal.
As an option, we can remove tests that check payload.